src/style.css

```diff
@@ -991,8 +991,8 @@ input:-webkit-autofill:focus {
     background-color: #fddfa3; /* color4 */
     -webkit-transition: 0.3s;
     transition: 0.3s;
-    -webkit-transform: translateY(0px);
-    transform: translateY(0px);
+    /* -webkit-transform: translateY(0px); */
+    /* transform: translateY(0px); */
```

Comment:
Any reason for commenting this out?

Reply:
This was creating problems in alumni-tiles component in single-blog class. Commenting it fixes that.

Reply:
Ok just make sure this doesn't have side effects on other components where it might be used.